addons/event/models/event_event.py

```diff
@@ -671,6 +671,13 @@ def _get_ics_file(self):
             result[event.id] = cal.serialize().encode('utf-8')
         return result
 
+    def _get_my_tickets_access_hash(self, registration_ids):
```

Comment:
Note that the name is misleading, it has no connection to "my" tickets, just some registrations.